wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -960,10 +964,10 @@ public void finalizeWipe() {
             @Override
             public void run() {
                 PlatformRepo.getInstance().clearDatabase(true);
-                ProcessPhoenix.triggerRebirth(WalletApplication.this);
```

Comment:
This should have been removed in a previous merge that resolved the Reset Wallet crash.  Though the location of this line was different than `master` at the time of the merge, so it would have been missed easily.